src/main/c/VipsImage.c

```diff
@@ -356,6 +363,30 @@ Java_com_criteo_vips_VipsImage_writePNGToArrayNative(JNIEnv *env, jobject obj, j
     return ret;
 }
 
+JNIEXPORT jbyteArray JNICALL
+Java_com_criteo_vips_VipsImage_writeAVIFToArrayNative(JNIEnv *env, jobject obj, jint q, jboolean lossless, jint speed)
```

Comment:
Same here, why is Vips providing a specific function for HEIF?

Reply:
This is like WritePNGToArrayNative, it's to inject optional arguments

Reply:
Ok, for this one. I misread it.